its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CodeCoverageTest.java

```diff
@@ -128,6 +128,7 @@ void whenAddingCoverage_ExclusionsAreRespected(
     var context = AnalysisContext.forServer("ExclusionsAndCoverage");
     context.begin.setDebugLogs();
     context.build.useDotNet();
+    context.end.setTimeout(2 * TestUtils.TIMEOUT_LIMIT);
```

Comment:
We could consider adding a `TestUtils.LARGE_TIMEOUT_LIMIT` instead.

Reply:
There's sometime 5 minutes, sometime 2 minutes. Should I make it like this?
```
Timeout.ONE_MINUTE (and use this as default)
Timeout.TWO_MINUTE
Timeout.FIVE_MINUTE
```